app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Vaccination certificate missing!")
+        if visitor["vaccine"]["expiration_date"] < date.today():
```

Comment:
It looks like u did't use `get` here for some purpose but I don't get it
Use same code style

Reply:
corrected